.ci/esy-build-steps.yml

```diff
@@ -12,21 +13,24 @@ steps:
   - script: npm run test
     displayName: "npm run test: before packing"
   - script: npm run package-cygwin
+    displayName: "Package cygwin"
+  - bash: npm pack
+    displayName: "npm pack"
   - task: PublishBuildArtifacts@1
     displayName: 'Release Package'
     inputs:
-        PathtoPublish: '.'
+        PathtoPublish: './esy-bash-0.3.20.tgz'
         ArtifactName: npm-package
   - script: node postinstall.js
     displayName: "node postinstall.js (iteration 1)"
   - script: npm run test
     displayName: "npm run test: after unpack (iteration 1)"
-  - script: node postinstall.js
-    displayName: "node postinstall.js (iteration 2)"
-  - script: npm run test
-    displayName: "npm run test: after unpack (iteration 2)"
-  - script: node postinstall.js
-    displayName: "node postinstall.js (iteration 3)"
-  - script: npm run test
-    displayName: "npm run test: after unpack (iteration 3)"
+  # - script: node postinstall.js
```

Comment:
Is this part correct?

Reply:
I have no idea why we iterate so many times. @bryphe Do you happen to remember why?

Reply:
I have no idea why we iterate so many times. @bryphe Do you happen to remember why?

Reply:
Was just unsure if it was meant to be here or something that was forgotten. This is fine by me :)